CONTRIBUTING.md

```diff
@@ -85,16 +85,19 @@ GLASS_ARRAY_BACKEND=jax python -m pytest
 GLASS_ARRAY_BACKEND=all python -m pytest
 ```
 
-Moreover, one can mark a test to be compatible with the array API standard by
-decorating it with `@array_api_compatible`. This will `parameterize` the test to
-run on every array library specified through `GLASS_ARRAY_BACKEND` -
+Moreover, one can test if a function is compatible with the array API standard
+by using the `xp` fixture. This will `parametrize` the test to run on every
+array library specified through `GLASS_ARRAY_BACKEND` -
 
 ```python
-import types
-from tests.conftest import array_api_compatible
+from __future__ import annotations
```

Comment:
Why do we need this?

Reply:
The import is required for the `TYPE_CHECKING` block. Removing the import gives -

```py
tests/conftest.py:16:12: TC004 Move import `types` out of type-checking block. Import is used for more than type hinting.
   |
15 | if TYPE_CHECKING:
16 |     import types
   |            ^^^^^ TC004
17 |
18 | from numpy.typing import NDArray
   |
   = help: Move out of type-checking block
```

See https://github.com/astral-sh/ruff/issues/7214 for more details.

Reply:
Ah, sorry, a bit confusing, I meant that `from __future__ import annotations` is required for the `TYPE_CHECKING` block to work. I get an error from ruff (attached above) if I remove `from __future__ import annotations`.

Reply:
Okay, understood. Just wonder if we could simplify the contributing guidelines with respect to typing, but that does kind of go against the point.